package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "hmpo-form-wizard": "5.1.1",
+    "hof-emailer": "^1.0.0",
```

Comment:
this is just bringing master up to date with so-master, so no changes have actually been made to this branch. I would suggest bumping hof-emailer in a separate PR